recipes/limitcpu/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.6" %}
+
+package:
+  name: limitcpu
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/limitcpu/files/limitcpu/cpulimit-{{ version }}.tar.gz
+  sha256: 2f8ceeb9de43a6de52ce50e67d618c715e714b72d806f3730b5661d5bfc9c075
+  patches:
+    - 0001-Fix-__APPLE_-typo.patch
```

Comment:
Sent the patch upstream to the e-mail address mentioned at http://limitcpu.sourceforge.net/contact.php.
The OSX build still failed to the function `getjiffies` being only defined for Linux and FreeBSD.